packages/obonode/obojobo-chunks-list/__snapshots__/editor-component.test.js.snap

```diff
@@ -3,15 +3,15 @@
 exports[`List Editor Node List component when not selected 1`] = `
 <div>
   <div
-    className="text-chunk obojobo-draft--chunks--list pad"
+    className="text-chunk obojobo-draft--chunks--list pad is-spacing-undefined"
   />
 </div>
 `;
 
 exports[`List Editor Node List component when selected 1`] = `
 <div>
   <div
-    className="text-chunk obojobo-draft--chunks--list pad"
+    className="text-chunk obojobo-draft--chunks--list pad is-spacing-undefined"
```

Comment:
I'd fix these undefineds - In the List editor-component.test.js file adding in the `spacing` prop for these two tests should fix the problem.